src/schema/v2/artwork/index.ts

```diff
@@ -861,6 +861,15 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return savedArtworkLoader(_id).then(({ is_saved }) => is_saved)
         },
       },
+      isDisliked: {
+        type: GraphQLBoolean,
```

Comment:
I'd mark this as non-nullable. We _never_ want a 3-state boolean (`true|false|null`). You can resolve with `!!is_disliked` to ensure.

Reply:
(Also I realize none of the other booleans are non-null: they should be — but don't worry about that with this PR)

Reply:
ok gotcha, gotta run through and add non-nullables on a bunch of data types for sure

Reply:
ok gotcha, gotta run through and add non-nullables on a bunch of data types for sure